UI/GamepadEmu.cpp

```diff
@@ -386,9 +386,12 @@ void InitPadLayout(float xres, float yres, float globalScale) {
 		g_Config.fUnthrottleKeyScale = scale;
 	}
 
-	//L and R------------------------------------------------------------
-	int l_key_X = 70 * scale;
-	int l_key_Y = 40 * scale;
+	// L and R------------------------------------------------------------
+	// Put them above the analog stick / above the buttons to the right.
+	// The corners were very hard to reach..
+
+	int l_key_X = 60 * scale;
+	int l_key_Y = yres - 110 * scale;
```

Comment:
Hmm, is this stuff related to the frame profiler or the other pull?

-[Unknown]